pkg/applicationmanager/controllers/applicationbackup.go

```diff
@@ -439,6 +439,32 @@ func (a *ApplicationBackupController) handle(ctx context.Context, backup *stork_
 		return nil
 	}
 
+	// if stork got restarted we would have got IncludeResource Memory map cleandup.
+	// hence re-create memory map of it.
+	if IsBackupObjectTypeVirtualMachine(backup) &&
+		backup.Status.Stage != stork_api.ApplicationBackupStageInitial &&
+		backup.Status.Stage != stork_api.ApplicationBackupStageImportResource &&
```

Comment:
Good to add ApplicationBackupStageFinal stage also as it stork gets restarted here we would end up populating in-memory content which is not needed.

Reply:
Do we need them in pre and post exec rule?

Reply:
For final stage its already taken care here 
https://github.com/libopenstorage/stork/blob/1559c07cc48ecb4a56c77d81c3418a67c30ed6b7/pkg/applicationmanager/controllers/applicationbackup.go#L362-L365

We want to ensure that we will do this for all stage except intial and for ImportResourceStage. So this works for preExec stage onwards.